### PR TITLE
Set the correct max stack size for spyglass

### PR DESCRIPTION
**Pull request by Gaprix**
## Introduction
Fixes the maximum stack size of spyglass

### API changes
Spyglass now has a separate class

### Behavioural changes
Now a player cannot have more than 1 spyglass in the slot

## Tests
```
/give PlayerName spyglass
```
before:
```
[PlayerName: Given Spyglass (spyglass) * 64 to PlayerName]
```
after:
```
[PlayerName: Given Spyglass (spyglass) * 1 to PlayerName]
```

### CODE REVIEW

File: src/item/ItemFactory.php [hunk 1]
```diff
@@ -98,6 +98,7 @@ public function __construct(){
 		$this->register(new GoldenApple(new IID(Ids::GOLDEN_APPLE), "Golden Apple"));
 		$this->register(new GoldenAppleEnchanted(new IID(Ids::ENCHANTED_GOLDEN_APPLE), "Enchanted Golden Apple"));
 		$this->register(new GoldenCarrot(new IID(Ids::GOLDEN_CARROT), "Golden Carrot"));
+		$this->register(new Spyglass(new IID(Ids::SPYGLASS), "Spyglass"));
```
Comment (dktapps) [inline src/item/ItemFactory.php]: Not clear why this is up here...

Comment (Gaprix) [reply src/item/ItemFactory.php]: Because of the alphabetical order of class names

Comment (dktapps) [reply src/item/ItemFactory.php]: This isn't in alphabetical order. S comes further down the alphabet.

Comment (Gaprix) [reply src/item/ItemFactory.php]: Oh, I realized my mistake. I'll fix it now

**Pull request merged**